docs/contributing.rst

```diff
@@ -115,10 +116,10 @@ Example: ``./test/libsolidity/syntaxTests/double_stateVariable_declaration.sol``
         uint128 variable;
     }
     // ----
-    // DeclarationError: Identifier already declared.
+    // DeclarationError: (36-52): Identifier already declared.
 
-A syntax test must contain at least the contract under test itself, followed by the separator ``----``. The additional comments above are used to describe the
-expected compiler errors or warnings. This section can be empty in case that the contract should compile without any errors or warnings.
+A syntax test must contain at least the contract under test itself, followed by the separator ``// ----``. The following comments are used to describe the
```

Comment:
Well, strictly speaking currently the separator is not necessary if the expected result is "no errors and no warnings" - in this case one can still use it, but it's optional - nevertheless I'd just keep it like this in the documentation - telling people to put the separator in there, even in this case, doesn't hurt.